### DISCUSSION TITLE
Combining Testing Data

### DISCUSSION THREAD

**Discussion by siddharth-tewari** [theOehrly/Fast-F1#115] [Q&A]
Is there a way to combine data for Day 1, 2 and 3 for some telemetry purposes?

**Comment by theOehrly** [comment]
I guess this depends on what you want to do. Most of FastF1's classes are built on top of Pandas. For example, you can use a function like ``pandas.concat`` to combine all laps of multiple days. This will allow you to find the fastest lap from multiple sessions and stuff like that. But you will not be able to get telemetry data for these laps any more, because the reference to each lap's associated telemetry data will be lost.

**Reply by siddharth-tewari** [reply]
Oh ok. My main objective was to combine all data so that I can get all laps and then the telemetry of them. Sadly its not possible as you said.
